Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/SingleFileTransfer.cs

```diff
@@ -199,7 +199,7 @@ private void PutDestinationFile(bool removeExisting = false)
     /// </summary>
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/e858f3196b2acf1842ce445ae12ddf23243a0a89/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/SingleFileTransfer.cs#L368
You can remove these cases which won't do anything, since default will handle those because there is no `break` in the cases.

Reply:
Removed